File: SIMauricio/PYTHON_TESTES/AULA02/Banco/ContaCorrente.py
from Conta import Conta
class ContaCorrente(Conta):
    def __init__(self, agencia, conta, saldo, qtdeCheques):
        super().__init__(agencia, conta, saldo)
        self.qtdeCheques = qtdeCheques

    def emitirCheques(self, qtde):
        if ((qtde >0) and (qtde <= self.qtdeCheques)):
            self.qtdeCheques = self.qtdeCheques - qtde
            return True
        else:
            return False

File: SIMauricio/JAVA_TESTES/AULA02/Banco/ContaCorrente.java
package Banco;

public class ContaCorrente extends Conta {
    public int qtdeCheques;

    //Construtores da Classe Filha
    ContaCorrente(){
    }
    
    ContaCorrente(String agencia, String conta, float saldo, int qtdeCheques){
        super(agencia, conta, saldo); //Chama o Construtor da Classe Mãe
        this.qtdeCheques = qtdeCheques;
    }

    public boolean emitirCheques(int qtde){
        if ((qtde >0) && (qtde <= qtdeCheques)){
            this.qtdeCheques = this.qtdeCheques - qtde;
            return true;
        }else{
            return false;
        }
    }

}
